deploy/helm/rawfile-localpv/values.yaml

```
@@ -142,6 +147,9 @@ node:
   # -- Number of gRPC workers for node component
   grpcWorkers: 10
 
+  # -- Port Number used for internal communication gRPC server
+  internalCommunicationPort: 4500
```

nit: or something else shorter
```suggestion
  internalGrpcPort: 4500
```

And we can take also the worker count from `grpcWorkers` ?